WLC/web_endpoint.py

```diff
@@ -6,12 +6,17 @@
 from flask import Flask, render_template
 from flask import request
 
-from WLC.code_executor.executor import CodeExecutor
-from WLC.image_processing.picture import Picture
+from .code_executor.executor import CodeExecutor
+from .image_processing.picture import Picture
 
 app = Flask(__name__)
 
 
+@app.route("/")
+def index():
+    return "Nothing to see here, this is just the API."
+
+
 @app.route("/api/upload_image", methods=['POST', 'GET'])
```

Comment:
Not sure if `upload_image` is the best name for the endpoint we can come up with. How about `execute` or something? (I am saying this because ideally, we should merge the `resubmit_code` function with this one).

Reply:
@RichardRohac